app/internal/event.py

```diff
@@ -84,3 +91,92 @@ def get_messages(session: Session,
         messages.append(f'Same event happened {weeks_diff} weeks before too. '
                         f'Want to create another one {weeks_diff} after too?')
     return messages
+
+
+def add_countries_to_db(session: Session) -> None:
+    """
+    Adding all new countries to the "Country" table in the database.
+    Information is based on the "countries" list.
+    (The list is located in app/resources/countries.py)
+    Names are described either as:
+    "Country Name, City Name" or
+    "Country Name" solely.
+    Timezones are described as "Continent/ City Name"
+    for example:
+        name: Israel, Jerusalem
+        timezone: Asia/Jerusalem
+    """
```

Comment:
The rationale was to allow maintainers to extend the countries list:)

Reply:
Are there any new countries coming soon? 🤔 JK but it is something that they can add to the dB

Reply:
:)
I am calling this function through 'get_all_countries_names' which is decorated by functools.lru_cache (by default it calls the inner part once in 124 times it will be called).
In addition according to your comment I added the following condition:
![image](https://user-images.githubusercontent.com/71094942/108580543-95951080-7334-11eb-9567-77e7b05860b0.png)